lib/Mojolicious/Plugin/OpenAPI.pm

```diff
@@ -25,6 +25,7 @@ sub register {
     $app->helper('openapi.valid_input' => sub { _validate($_[0]) ? undef : $_[0] });
     $app->helper('openapi.spec'        => \&_helper_spec);
     $app->helper('reply.openapi'       => \&_reply);
+    $app->helper('openapi.not_implemented' => $config->{not_implemented} || \&NOT_IMPLEMENTED);
```

Comment:
There's no reason for this. The user can just override the helper from the app after loading the plugin.